coalib/bears/Bear.py

```diff
@@ -26,6 +26,14 @@
 
 class Debugger(pdb.Pdb):
 
+    def __init__(self, *args, bear=None, **kwargs):
```

Comment:
since `bear` is actually not optional, you should remove the default value `None`